src/state-item.c

```diff
@@ -0,0 +1,417 @@
+//
+//  state-item.c
+//  bison
+//
+//  Created by Vincent Imbimbo on 10/15/19.
+//  Copyright © 2019 Vincent Imbimbo. All rights reserved.
+//
+
+#include <config.h>
+#include <stdlib.h>
+#include "state-item.h"
+#include "closure.h"
+#include "nullable.h"
+#include <time.h>
+#include <gl_linked_list.h>
+#include <gl_xlist.h>
+#include <assert.h>
+#include "getargs.h"
```

Comment:
Please, first config.h (and possibly "system.h"), then <> headers (including those of gnulib), and finally those in src/ as "".

See how it's done in the existing files.